Util/AnnotationLimitProcessor.php

```diff
@@ -44,14 +44,18 @@ public function getRateLimit(Request $request)
         return $best_match;
     }
 
-    public function getRateLimitAlias(Request $request)
+    public function getRateLimitAlias(Request $request, callable $controller)
```

Comment:
Can't we use `$request->attributes->get('_controller')` as `$controller` ?

Reply:
That's definitely an option! That would be great